src/igvfd/tests/fixtures/schemas/primary_cell.py

```diff
@@ -8,19 +8,21 @@ def primary_cell(testapp, other_lab, award, human_donor, sample_term_pluripotent
         'lab': other_lab['@id'],
         'source': other_lab['@id'],
         'donors': [human_donor['@id']],
-        'biosample_term': sample_term_pluripotent_stem_cell['@id']
+        'biosample_term': sample_term_pluripotent_stem_cell['@id'],
+        'virtual': True
```

Comment:
why do you make primary_cell virtual?

Reply:
This is needed to support my unit test named: test_non_virtual_sample_linked_to_virtual_sample_with_single_property

The fixture primary_cell_with_part_of_virtual_true has part_of linked to the fixture primary_cell.  primary_cell_with_part_of_virtual_true is set to virtual False.  primary_cell is set to virtual True.

Reply:
this fixture is being used by other tests, are you sure they are ok with it becoming virtual?

Reply:
this fixture is being used by other tests, are you sure they are ok with it becoming virtual?

Reply:
All unit tests passed with this change, but I can make another fixture if you want?

Reply:
I have removed the virtual property from the primary_cell fixture and patched it in the unit test instead.